slices/ca-certificates-java.yaml

```diff
@@ -0,0 +1,33 @@
+package: ca-certificates-java
+
+essential:
+  - ca-certificates-java_copyright
+
+slices:
+  bins:
+    essential:
+      - ca-certificates_bins
+      - ca-certificates_data-with-certs
+      # chisel tool does not allow per-architecture packages
+      # openjdk-8 is not published for RISCV in the archive
+      # - openjdk-8-jre-headless_security
```

Comment:
I think the dependency should be the other way around? I mean, `ca-certificates-java` should not depend on `openjdk-..`, rather `openjdk-..` should depend on `ca-certificates-java`?

Reply:
This is  a chicken-and-egg packaging issue. 
ca-certificate-java used to have circular dependency with openjdk. 
The current state openjdk depends on ca-certificates-java and maintainer script checks if it can find java in the common locations. 

In the chisel story it does not make sense to make ca-certificates-java a dependency of openjdk - we do not need maintainer script installed in the chiselled rootfs unless we intend to update cacerts in place as per instructions below. 

Normal use-case would have something like https://github.com/canonical/rockcraft/pull/688/files#diff-e14fbacbe2674da891b0458de834490fc20512c6f00379284b659fce6ef0a927 where we copy the cacerts from the build host.